parsl/dataflow/dflow.py

```diff
@@ -409,7 +409,7 @@ def launch_task(self, task_id, executable, *args, **kwargs):
             executable = app_monitor.monitor_wrapper(executable, task_id, self.monitoring_config, self.run_id)
         exec_fu = executor.submit(executable, *args, **kwargs)
         self.tasks[task_id]['status'] = States.running
-        self.tasks[task_id]['time_started'] = str(datetime.now().strftime('%Y-%m-%d %H:%M:%S'))
+        self.tasks[task_id]['time_submitted'] = time.time()
```

Comment:
Can we move this `time.time()` call to the very start of this method? In my view, `time_submitted` should be the time the _user_ submitted the task. For measuring latency we want to be able to measure how long it takes us to go round trip (user submits task -> user receives task output).